### ISSUE TITLE
archive cleanup: move W01CT0060 eTexts

### ISSUE THREAD

**Issue by TBRC-Travis** [BuddhistDigitalResourceCenter/archive-issues#7]
Not sure where W01CT0060/archive/texts should go.  maybe eTextsArchive?

**Comment by xristy** [comment]
Your suggestion is good, except I'd like to start using `Sources` rather than `archive` lingo for the simple reason that the word archive is being used in at least two ways and I don't think it's so clear for others who aren't enmeshed in the day-to-day use of the resources.

So I would suggest `eTextSources`.

It turns out that for all of the eTexts that I've worked with I've kept the original source files (`*.dct` and `*.doc` primarily) and these could be folded in to the `Sources`.

Along with the original source files I have also kept the intermediate `*.rtf` files that were derived from the `*.dct` and `*.doc` files. These file set should also be considered for inclusion in the _archived_ `Sources` perhaps under `eTextRTFs`.

So perhaps the eventual structure at S3/Glacier would be:

    archive.tbrc.org/Sources/{Images, eTextSources, eTextRTFs}

On RS3/4 for now we can just the dirs directly under the Work dir:

    W01CT0060/{archive, eTextSources, eTextRTFs, eBooks, images, prints}
as appropriate.

I realize that you don't have the eTextRTFs for any of the works at this point and that we'll have to make a task to deal with integrating these items at a later point.
